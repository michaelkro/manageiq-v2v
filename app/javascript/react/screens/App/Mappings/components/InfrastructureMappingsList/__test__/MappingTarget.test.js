import React from 'react';
import { shallow } from 'enzyme';

import MappingTarget from '../components/MappingTarget';

test('renders MappingTarget with children', () => {
  const wrapper = shallow(
    <MappingTarget>
      <div>hello</div>
    </MappingTarget>
  );

  expect(wrapper).toMatchSnapshot();
});
