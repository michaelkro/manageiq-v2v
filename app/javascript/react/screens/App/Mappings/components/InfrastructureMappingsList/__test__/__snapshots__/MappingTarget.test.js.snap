// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MappingTarget with children 1`] = `
<div
  className="infra-mapping-item-target"
>
  <div>
    hello
  </div>
</div>
`;
